Dockerfile

```diff
@@ -60,9 +59,17 @@ WORKDIR /app
 
 ENV SHELL=bash
 RUN cd /app && git clone https://github.com/yuanming-hu/taichi.git
-RUN cd /app/taichi && python3.6 dev_setup.py
-RUN cd /app/taichi && mkdir build && cd build && cmake .. -DPYTHON_EXECUTABLE=python3.6
-RUN cd /app/taichi/build && make -j `nproc --all`
+RUN mkdir /app/taichi/build
+WORKDIR /app/taichi/build
+RUN export CUDA_BIN_PATH=/usr/local/cuda-10.0 && \
+    cmake .. -DPYTHON_EXECUTABLE=$(which python3.6) -DCUDA_VERSION=10.0 -DTLANG_WITH_CUDA:BOOL="True" && \
+    make -j 15 && \
+    ldd libtaichi_core.so
+
+WORKDIR /app/taichi/python
+ENV PATH="/app/taichi/bin:$PATH"
```

Comment:
Please add after this line:
```
ENV TAICHI_REPO_DIR="/app/taichi"
```
This will probably solve your `libtaichi_core.so` path issue.